Reflection_Engine/Query/InheritedTypes.cs

```diff
@@ -39,7 +39,7 @@ public static List<Type> InheritedTypes(this Type type, bool onlyBHoM = false)
         {
             List<Type> types = type.GetInterfaces().ToList();
 
-            if (type.BaseType != typeof(object))
+            if (type.BaseType != null && type != typeof(object))
```

Comment:
Shouldn't it be `(type.BaseType != null && type.BaseType != typeof(object))` ?

Reply:
Of course it should be 🤦 I have just committed a fix, thanks for being careful!